dep0/lib/03_typecheck/src/private/returns_from_all_branches.hpp

```diff
@@ -4,6 +4,13 @@
 
 namespace dep0::typecheck {
 
+/**
+ * Check whether the given body contains an explicit return statement
+ * from every branch.
+ *
+ * @return True if all branches contain an explicit return statement,
+ *         false otherwise.
```

Comment:
```suggestion
 * Checks whether the given body contains an explicit return statement from every branch.
 *
 * @return True if all branches contain an explicit return statement, false otherwise.
```